src/osx/cocoa/dataview.mm

```diff
@@ -3565,13 +3565,18 @@ - (NSPoint)cellBaselineOffset
         m_renderer->OSXUpdateAlignment();
 }
 
-void wxDataViewColumn::SetBitmap(const wxBitmap& bitmap)
+void wxDataViewColumn::SetBitmap(const wxBitmapBundle& bitmap)
 {
     // bitmaps and titles cannot exist at the same time - if the bitmap is set
     // the title is removed:
     m_title.clear();
     wxDataViewColumnBase::SetBitmap(bitmap);
-    [[m_NativeDataPtr->GetNativeColumnPtr() headerCell] setImage:bitmap.GetNSImage()];
+    wxBitmap bmp = m_owner ? bitmap.GetBitmapFor(m_owner) : bitmap.GetBitmap(
+        bitmap.GetPreferredBitmapSizeAtScale(
+            wxOSXGetMainScreenContentScaleFactor()
+        )
+    );
+    [[m_NativeDataPtr->GetNativeColumnPtr() headerCell] setImage:bmp.GetNSImage()];
```

Comment:
I think the reason that this doesn't work (and didn't work even before these changes) is that the comment above about the cell displaying either text or bitmap but not both is correct, but clearing the text isn't really enough to display the image. We probably need to reallocate the cell if it contains text currently, i.e. use `initImageCell:` to create a new cell and assign it to `headerCell`.

But this behaviour is not really useful anyhow, I don't think we often want to show just the bitmap, without any text. Does anybody know if it's possible to show both at the same time in a single `NSCell` somehow? I.e. is there some subclass of it capable of it? 

@csomor This is not that important but if, by chance, you have any pointers here, this would be very helpful.